ubclaunchpad/ubc-degree-navigator
# Set up routing between pages

## Describe your changes
> Add a short description of the changes. These will be the things to check for when the reviewers review your PR.

The frontend should look exactly as it did before LOL. 

I changed the underlying structure of the frontend codebase: previously, there was one component (MultiStepForm.jsx) that displayed the app. This component used a `switch` statement to change what was being displayed. Now, there are separate components/pages for each action the user can take, and react-router is used to move between pages. 

#### Screenshot (if applicable)
> This can be FE screenshot, DB screenshot, API call from network tab, etc.

## How do I test
> Provide brief steps on how to test these changes. 
1. Go through the steps of uploading your course history, program information, etc.
2. Ensure that each button (previous and next) takes you to the correct page and is enabled at the correct times
3. Check that the styling looks as it did before as well

## Checklist before requesting a review
- [ ] My base and comparing branches are correct
